ipfs-cluster-ctl/main.go

```diff
@@ -45,7 +46,9 @@ specific one.
 %s uses the IPFS Cluster API to perform requests and display
 responses in a user-readable format. The location of the IPFS
 Cluster server is assumed to be %s, but can be
-configured with the --host option.
+configured with the --host option. To use the secure libp2p-http
+API endpoint, use "--host" with the full cluster libp2p listener
+address (including the "/ipfs/<peerID>" part), and --secret.
```

Comment:
It could be a bit clearer that `--secret` is referring to the `CLUSTER_SECRET`.

Reply:
It's in the help for the flag, but added explanation.